_maps/_basemap.dm

```diff
@@ -1,4 +1,4 @@
-//#define LOWMEMORYMODE //uncomment this to load centcom and runtime station and thats it.
+#define LOWMEMORYMODE //uncomment this to load centcom and runtime station and thats it.
```

Comment:
you don't need to uncomment this - vscode has a low-memory mode variants for all of the build/run tasks

Reply:
thats unintentionally left in, oops
and also thats a THING??? MAAAN